package.json

```diff
@@ -20,8 +20,25 @@
   "license": "MIT",
   "devDependencies": {
     "autoprefixer": "^10.3.1",
+    "eslint": "^7.31.0",
+    "eslint-config-next": "^11.0.1",
+    "eslint-config-prettier": "^8.3.0",
+    "faker": "^5.5.3",
+    "husky": "^7.0.1",
+    "lint-staged": "^11.0.1",
     "postcss": "^8.3.5",
+    "prettier": "^2.3.2",
+    "pretty-quick": "^3.1.1",
     "prisma": "2.27.0",
+    "slugify": "^1.6.0",
     "tailwindcss": "^2.2.4"
+  },
+  "lint-staged": {
```

Comment:
Lint-staged is a tool that allows us to check only the staged files, so we don't have to format/check the whole codebase each time.